assets/beach/beach.level.yaml

```diff
@@ -68,6 +68,10 @@ enemies:
   - fighter: *brute
     location: [450, 20, 0]
     trip_point_x: 300
+  - fighter: *brute
+    location: [1000, 20, 0]
```

Comment:
Does this brute need a trip point specified?

Reply:
Right! It got lost in a rebase conflict  :sweat_smile:

Reply:
I've been there :smile: 

I've done a lot of ugly rebases in the past.